test/e2e/gardener/shoot/create_hibernate_wakeup_delete.go

```diff
@@ -35,6 +36,8 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 		Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
 		f.Verify()
 
+		node.VerifyNodeCriticalComponentsBootstrapping(ctx, f.ShootFramework)
```

Comment:
Why did you extend specifically this test? Doesn't this better fit into the default test?

Reply:
We extended this test because it only uses a single Node, while the default test deals with multiple nodes. Deleting all of them and waiting for them to be recreated might increase the test duration undesirably.

Reply:
Code comment :)